package.json

```diff
@@ -40,11 +40,11 @@
     "semantic-release-gitmoji": "^1.3.4",
     "tap-difflet": "^0.7.2",
     "tape": "^4.10.1",
-    "webpack": "^4.41.5",
-    "webpack-cli": "^3.3.10"
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0"
   },
   "peerDependencies": {
-    "webpack": "^4.0.0"
+    "webpack": "^5.74.0"
```

Comment:
I think this would be `"4.x || 5.x"`, unless it's actually stopped working on Webpack 4.